primitives/src/condition.rs

```diff
@@ -61,26 +123,32 @@ pub struct Condition {
     #[migrate]
     pub condition_type: ConditionType,
     /// Trusted issuers.
-    pub issuers: Vec<IdentityId>,
+    #[migrate(TrustedIssuer)]
+    pub issuers: Vec<TrustedIssuer>,
 }
 
 #[allow(missing_docs)]
 impl Condition {
     /// Generate condition on the basis of `condition_type` & `issuers`.
-    pub fn new(condition_type: ConditionType, issuers: Vec<IdentityId>) -> Self {
+    pub fn new(condition_type: ConditionType, issuers: Vec<TrustedIssuer>) -> Self {
         Self {
             condition_type,
             issuers,
         }
     }
+
+    /// Create a condition with the given type and issuers trusted for any claim type.
+    pub fn from_dids(condition_type: ConditionType, issuers: &[IdentityId]) -> Self {
```

Comment:
Maybe impl `From<(ConditionType, &[IdentityId])>`?

Reply:
I did consider your suggestion while writing the code. However, with `From`s used just for the sake of `.into()` rather than to satisfy a bound, I think you need to be careful not to write too obscure and opaque code. A single type is imo a fair balance, but taking a tuple goes beyond what I think is readable, so I prefer to avoid `(ty, ids).into()`.